internal/command/deploy/deploy.go

```diff
@@ -365,19 +378,29 @@ func deployToMachines(
 		return err
 	}
 
-	excludeRegions := make(map[string]interface{})
-	for _, r := range flag.GetStringSlice(ctx, "exclude-regions") {
-		reg := strings.TrimSpace(r)
-		if reg != "" {
-			excludeRegions[reg] = struct{}{}
-		}
+	excludeRegions := make(map[string]bool)
```

Comment:
`map[string]bool` is slightly wasteful compared to `map[string]interface{}`, but the ergonomics are so much better for this kind of work.